### ISSUE TITLE
Re-implement Cassandra source

### ISSUE THREAD

**Issue by adriancole** [openzipkin/zipkin-dependencies-hadoop#2]
when I try execute zipkin-dependencies-hadoop
```bash
hadoop jar build/libs/zipkin-dependencies-hadoop-*-all.jar --source cassandra --hosts localhost --port 9160
```
return me:
```bash
Exception in thread "main" java.io.IOException: Mkdirs failed to create /var/folders/75/b_p3cc112y52tvjdfg1bkz580000gp/T/hadoop-unjar2090021427468948969/META-INF/license
	at org.apache.hadoop.util.RunJar.ensureDirectory(RunJar.java:128)
	at org.apache.hadoop.util.RunJar.unJar(RunJar.java:104)
	at org.apache.hadoop.util.RunJar.unJar(RunJar.java:81)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:209)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
```
I have read that it's problem of macos. I try to execute from debian but:
```
root@debian0:~# /root/hadoop-2.7.1/bin/hadoop jar /root/zipkin-aggregate-twitter-is-here-all.jar --source cassandra --hosts 172.16.144.1 --port 9160
Exception in thread "main" java.lang.Throwable: If you know what exactly caused this error, please consider contributing to GitHub via following link.
https://github.com/twitter/scalding/wiki/Common-Exceptions-and-possible-reasons#comtwitterscaldingargsexception
	at com.twitter.scalding.Tool$.main(Tool.scala:130)
	at com.twitter.zipkin.aggregate.RunHadoopJob$delayedInit$body.apply(RunHadoopJob.scala:4)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:40)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:71)
	at scala.App$$anonfun$main$1.apply(App.scala:71)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:32)
	at scala.App$class.main(App.scala:71)
	at com.twitter.zipkin.aggregate.RunHadoopJob$.main(RunHadoopJob.scala:3)
	at com.twitter.zipkin.aggregate.RunHadoopJob.main(RunHadoopJob.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:221)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.twitter.scalding.Job$.apply(Job.scala:45)
	at com.twitter.scalding.Tool.getJob(Tool.scala:46)
	at com.twitter.scalding.Tool.run(Tool.scala:66)
	at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:70)
	at com.twitter.scalding.Tool$.main(Tool.scala:126)
	... 16 more
Caused by: com.twitter.scalding.ArgsException: cassandra is not an implemented source.
	at com.twitter.zipkin.aggregate.SpanSourceProvider$.apply(ZipkinAggregateJob.scala:54)
	at com.twitter.zipkin.aggregate.ZipkinAggregateJob.<init>(ZipkinAggregateJob.scala:16)
	... 25 more
```

**Comment by adriancole** [comment]
Here's the commit to resurrect files from: 6e8fdad#diff-9e4ef735c2cc6a6622c3f51167c24b80

The hadoop/scalding code for cassandra goes here. The Aggregates code will be completed as a part of https://github.com/openzipkin/zipkin/pull/599

**Comment by 3thinkthendoit** [comment]
i get the same problem:
Caused by: java.lang.reflect.InvocationTargetException
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
    at com.twitter.scalding.Job$.apply(Job.scala:45)
    at com.twitter.scalding.Tool.getJob(Tool.scala:46)
    at com.twitter.scalding.Tool.run(Tool.scala:66)
    at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:70)
    at com.twitter.scalding.Tool$.main(Tool.scala:126)
    ... 16 more
Caused by: com.twitter.scalding.ArgsException: cassandra is not an implemented source.
    at com.twitter.zipkin.aggregate.SpanSourceProvider$.apply(ZipkinAggregateJob.scala:54)
    at com.twitter.zipkin.aggregate.ZipkinAggregateJob.<init>(ZipkinAggregateJob.scala:16)
    ... 25 more

**Comment by suowik** [comment]
I got the same stacktrace:

Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at com.twitter.scalding.Job$.apply(Job.scala:45)
	at com.twitter.scalding.Tool.getJob(Tool.scala:50)
	at com.twitter.scalding.Tool.run(Tool.scala:70)
	at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:70)
	at com.twitter.scalding.Tool$.main(Tool.scala:150)
	... 11 more
Caused by: com.twitter.scalding.ArgsException: cassandra is not an implemented source.
	at com.twitter.zipkin.dependencies.SpanSourceProvider$.apply(ZipkinDependenciesJob.scala:51)
	at com.twitter.zipkin.dependencies.ZipkinDependenciesJob.<init>(ZipkinDependenciesJob.scala:16)
	... 20 more

**Comment by adriancole** [comment]
FYI everyone will get this, so no need to prepare the same exception.

**Comment by 3thinkthendoit** [comment]
@adriancole 

i cloned the https://github.com/openzipkin/zipkin-dependencies-hadoop project ,
and set the branch for msater. then gradlew.bat run on the windows . 

but i don't how to solve this exception ? thanks! hehehe.

**Comment by eirslett** [comment]
@3thinkthendoit The problem is that the Cassandra implementation was removed when we moved away from the Cassie Cassandra driver to the Datastax driver. Reimplementing the Hadoop job using the Datastax driver is still work in progress.
TL;DR the dependencies/aggregate stuff doesn't work nowadays.

**Comment by adriancole** [comment]
;)
